v4/source/status.yaml

```diff
@@ -16,3 +21,44 @@
           $ref: '#/responses/BadRequest'
         '401':
           $ref: '#/responses/Unauthorized'
+    put:
+      tags:
+        - status
+      summary: Update user status
+      description: |
+        Update user status.
```

Comment:
Can we change this to `Manually set a user's status. When setting a user's status, the status will remain that value until set "online" again, which will return the status to being automatically updated based on user activity.`